StructureDefinition/NHSDigital-ServiceRequest-Genomics.StructureDefinition.json

```diff
@@ -174,6 +164,14 @@
         "definition": "A code that identifies a particular service (i.e., procedure, diagnostic investigation, or panel of investigations) that have been requested.\r\n\r\nFor Genomic Test Orders, use the system 'https://www.england.nhs.uk/publication/national-genomic-test-directories' and code from one of the test directories provided by NHS England. Addition of these codes to an appropriate coding system, e.g. SNOMED-CT, is under discussion.",
         "min": 1
       },
+      {
+        "id": "ServiceRequest.orderDetail",
```

Comment:
While you are profiling this, could you add another slice for the CodeSystem URL defined in NamingSystem-England-GenomicTestDirectory (to allow Test Directory Codes to also be used here without warnings)